tests/build/build.go

```diff
@@ -61,10 +61,15 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 				Skip("Using private cluster (not reachable from Github), skipping...")
 			}
 
+			quayOrg := utils.GetEnv("DEFAULT_QUAY_ORG", "")
```

Comment:
Should we handle the case when `DEFAULT_QUAY_ORG` is not set?

Reply:
I don't think so, since it is mandatory to set `DEFAULT_QUAY_ORG` when running the e2e-tests as defined [here](https://github.com/redhat-appstudio/e2e-tests/blob/main/default.env#L13) .. the test will fail anyway if this env var is not set

Reply:
If the env var is not set, is the test skipped with message "Quay org does not support privat..." ?

By reading the code, seems no need to pass the default value explicitly. If that env var is not set, an empty string wll be returned. Am I right? :)

Reply:
We have preflight check in place, if we don't set the env `DEFAULT_QUAY_ORG`, during cluster boot strapping will get error: `Error: error when running preflight checks: required env vars containing secrets (DEFAULT_QUAY_ORG) not defined or empty`